docs/source/contributing/docs-contributions/doc-new-repos.md

```diff
@@ -29,7 +29,7 @@ demo notebooks, if available.
 
 The *Resources* section includes:
 
-![Resources section in `README.md`](static/repo-resources.png "Screenshot of resource list at end of README file")
+![Resources section in `README.md` ](static/repo-resources.png "Screenshot of resource list at end of README file")
```

Comment:
```suggestion

```
## Resources
- [Project Jupyter website](https://jupyter.org)
- [Online Demo at jupyter.org/try](https://jupyter.org/try)
- [Documentation for Jupyter notebook](https://jupyter-notebook.readthedocs.io/en/latest/) [[PDF](https://media.readthedocs.org/pdf/jupyter-notebook/latest/jupyter-notebook.pdf)]
- [Korean Version of Installation](https://github.com/ChungJooHo/Jupyter_Kor_doc/)
- [Documentation for Project Jupyter](https://jupyter.readthedocs.io/en/latest/index.html) [[PDF](https://media.readthedocs.org/pdf/jupyter/latest/jupyter.pdf)]
- [Issues](https://github.com/jupyter/notebook/issues)
- [Technical support - Jupyter Google Group](https://groups.google.com/forum/#!forum/jupyter)
```
```